ACPs/77-reinventing-subnets/README.md

```diff
@@ -364,50 +372,50 @@ type IncreaseSubnetValidatorBalanceTx struct {
 }
 ```
 
-If the Subnet Validator corresponding to `ValidationID` is currently inactive (`Balance` was exhausted or `DisableSubnetValidatorTx` was issued), this transaction will move them back to the active validator set.
+If the validator corresponding to `ValidationID` is currently inactive (`Balance` was exhausted or `DisableL1ValidatorTx` was issued), this transaction will move them back to the active validator set.
 
-Note: The $AVAX added to `Balance` can be claimed at any time by the Subnet Validator using `DisableSubnetValidatorTx`.
+Note: The $AVAX added to `Balance` can be claimed at any time by the validator using `DisableL1ValidatorTx`.
 
-### Bootstrapping Subnet Nodes
+### Bootstrapping L1 Nodes
 
 Bootstrapping a node/validator is the process of securely recreating the latest state of the blockchain locally. At the end of this process, the local state of a node/validator must be in sync with the local state of other virtuous nodes/validators. The node/validator can then verify new incoming transactions and reach consensus with other nodes/validators.
 
 To bootstrap a node/validator, a few critical questions must be answered: How does one discover peers in the network? How does one determine that a discovered peer is honestly participating in the network?
 
 For standalone networks like the Avalanche Primary Network, this is done by connecting to a hardcoded [set](https://github.com/ava-labs/avalanchego/blob/master/genesis/bootstrappers.json) of trusted bootstrappers to then discover new peers. Ethereum calls their set [bootnodes](https://ethereum.org/developers/docs/nodes-and-clients/bootnodes).
 
-By separating Subnet Validators from Primary Network Validators, a list of validator IPs to connect to (the functional bootstrappers of the Subnet) is no longer provided by simply connecting to the Primary Network Validators. However, the Primary Network can enable nodes tracking a Subnet to seamlessly connect to the Subnet Validators by tracking and gossiping Subnet Validator IPs. Subnets will not need to operate and maintain a set of bootstrappers and can continue to rely on the Primary Network for peer discovery.
+By separating L1 validators from Primary Network validators, a list of validator IPs to connect to (the functional bootstrappers of the L1) is no longer provided by simply connecting to the Primary Network validators. However, the Primary Network can enable nodes tracking an L1 to seamlessly connect to the validators by tracking and gossiping validator IPs. L1s will not need to operate and maintain a set of bootstrappers and can continue to rely on the Primary Network for peer discovery.
```

Comment:
```suggestion
Since L1 validators are not required to be Primary Network validators, a list of validator IPs to connect to (the functional bootstrappers of the L1) cannot be provided by simply connecting to the Primary Network validators. However, the Primary Network can enable nodes tracking a L1 to seamlessly connect to the validators by tracking and gossiping L1 validator IPs. L1s will not need to operate and maintain a set of bootstrappers and can rely on the Primary Network for peer discovery.
```

Reply:
https://github.com/avalanche-foundation/ACPs/pull/154/files#r1816820001 same change excluding `a L1`